xbmc/guilib/GUIComponent.cpp

```diff
@@ -20,13 +20,17 @@
 
 #include "GUIComponent.h"
 #include "GUIWindowManager.h"
+#include "TextureManager.h"
 #include "dialogs/GUIDialogYesNo.h"
+#include "GUILargeTextureManager.h"
 #include "ServiceBroker.h"
 #include "URL.h"
 
 CGUIComponent::CGUIComponent()
 {
   m_pWindowManager.reset(new CGUIWindowManager());
+  m_pTextureManager.reset(new CGUITextureManager());
+  m_pLargeTextureManager.reset(new CGUILargeTextureManager());
```

Comment:
if `m_pWindowManager->Initialize()` doesn't depend on these then they should happen after m_pWindowManager is initialized

Reply:
I dropped m_pWindowManager->Initialize() from ctor. Putting behaviour in ctor is not a good style anyway.